packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/modals/EditPlanPreserveStaticIPs/EditPlanPreserveStaticIPs.tsx

```diff
@@ -71,10 +68,10 @@ const EditPlanPreserveStaticIPs_: React.FC<EditPlanPreserveStaticIPsProps> = (pr
       {...props}
       jsonPath={(obj: V1beta1Plan) => (obj.spec.preserveStaticIPs ? 'true' : 'false')}
       title={props?.title || t('Set to preserve the static IPs')}
-      label={props?.label || t('Whether to preserve the static IPs')}
+      label={props?.label ?? ''}
       model={PlanModel}
+      body={<br />}
```

Comment:
nitpick: Having a break-line as the only content in the body of a modal is a little odd IMO. Is there another way we can space items (assuming this was the reason) that is more straightforward?

Reply:
So let's avoid this spacing addition.  `title`, `label` fields are strings and I don't want to touch the `EditModal` comp, But it i not crucial anyway, it looks ok even without this space line.